packages/evolution-backend/src/services/audits/auditChecks/checks/README.md

```
@@ -4,52 +4,36 @@ This directory contains the actual audit check functions organized by survey obj
 
-All audit check functions follow the pattern: `[SURVEY_OBJECT]_[TYPE]_[Description]`
-
-- **Entity prefixes**: `I` (Interview), `HH` (Household), `HM` (Home), `P` (Person), `J` (Journey), `VP` (VisitedPlace), `T` (Trip), `S` (Segment)
-- **Audit types**: `I` (Invalid), `L` (Logical), `M` (Missing)
-
-## Current Audit Checks
-
-### Missing Data Checks (M)
+All audit check functions follow the pattern: `[OBJECTPREFIX]_[AUDITCHECKTYPE]_[Description]`
+
+### Survey Object Prefixes
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)